src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/CodingComponent.tsx

```diff
@@ -23,9 +23,13 @@ const getValueSetComposeIncludeOids = (valueSet: ValueSet) => {
   return valueSet.compose.include
     .map((include) => include.valueSet)
     .reduce((acc, curr) => {
-      const oid = getOidFromString(curr[0], "FHIR");
+      const oid = getOidFromString(curr?.[0], "FHIR");
       if (oid) {
-        return [...acc, oid];
+        const combined = [...acc, oid];
+        // Remove duplicates
+        return combined.filter(
+          (item, index, self) => index === self.findIndex((vs) => vs === item)
+        );
```

Comment:
I'm not sure this removes duplicates. Is there a unit test?

Reply:
@jkotanchik-SB This was an earlier change, it would remove the duplicate, e.g. OMB Race Categories, but, it will still show even when a user clicks on Ethnicity. It's not needed any more, I added some unit tests.